
Plotting labels...
Scanning '/home/cx/rmvision/datasets/final/train/labels.cache' for images and labels... 180 found, 0 missing, 0 empty, 0 corrupted: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████| 180/180 [00:00<?, ?it/s]
Scanning '/home/cx/rmvision/datasets/final/val/labels.cache' for images and labels... 20 found, 0 missing, 0 empty, 0 corrupted: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:00<?, ?it/s]
None
Traceback (most recent call last):
  File "train.py", line 515, in <module>
    train(hyp, opt, device, tb_writer, wandb)
  File "train.py", line 210, in train
    plot_labels(labels, save_dir, loggers)
  File "/home/cx/rmvision/yolov5-RM/utils/plots.py", line 266, in plot_labels
    sns.pairplot(x, corner=True, diag_kind='auto', kind='hist', diag_kws=dict(bins=50), plot_kws=dict(pmax=0.9))
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/seaborn/_decorators.py", line 46, in inner_f
    return f(**kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/seaborn/axisgrid.py", line 2096, in pairplot
    grid = PairGrid(data, vars=vars, x_vars=x_vars, y_vars=y_vars, hue=hue,
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/seaborn/_decorators.py", line 46, in inner_f
    return f(**kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/seaborn/axisgrid.py", line 1244, in __init__
    axes = fig.subplots(len(y_vars), len(x_vars),
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/_api/deprecation.py", line 471, in wrapper
    return func(*args, **kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/figure.py", line 911, in subplots
    axs = gs.subplots(sharex=sharex, sharey=sharey, squeeze=squeeze,
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/gridspec.py", line 307, in subplots
    axarr[row, col] = figure.add_subplot(
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/figure.py", line 784, in add_subplot
    ax = subplot_class_factory(projection_class)(self, *args, **pkw)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axes/_subplots.py", line 36, in __init__
    self._axes_class.__init__(self, fig, [0, 0, 1, 1], **kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/_api/deprecation.py", line 471, in wrapper
    return func(*args, **kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axes/_base.py", line 634, in __init__
    self.cla()
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axes/_base.py", line 1296, in cla
    self.xaxis.set_clip_path(self.patch)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axis.py", line 919, in set_clip_path
    for child in self.majorTicks + self.minorTicks:
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axis.py", line 612, in __get__
    tick = instance._get_tick(major=False)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axis.py", line 2059, in _get_tick
    return XTick(self.axes, 0, major=major, **tick_kw)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/axis.py", line 432, in __init__
    self.gridline.set(
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/artist.py", line 1154, in set
    kwargs = cbook.normalize_kwargs(kwargs, self)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/_api/deprecation.py", line 431, in wrapper
    return func(*inner_args, **inner_kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/_api/deprecation.py", line 431, in wrapper
    return func(*inner_args, **inner_kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/matplotlib/_api/deprecation.py", line 406, in wrapper
    arguments = signature.bind(*inner_args, **inner_kwargs).arguments
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/inspect.py", line 3037, in bind
    return self._bind(args, kwargs)
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/inspect.py", line 2997, in _bind
    param_name = param.name
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/inspect.py", line 2539, in name
    @property
  File "/home/cx/miniconda3/envs/ml/lib/python3.8/site-packages/torch/utils/data/_utils/signal_handling.py", line 66, in handler
    _error_if_any_worker_fails()
RuntimeError: DataLoader worker (pid 42379) exited unexpectedly with exit code 1. Details are lost due to multiprocessing. Rerunning with num_workers=0 may give better error trace.
None
None
None